src/components/Toolbar/ToolbarRightContent.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+const ToolbarRightContent = ({ children, className, ...rest }) => {
+  const classes = cx('toolbar-pf-action-right', className);
```

Comment:
do we want the ```...rest``` passed through to the div?

Reply:
yes, added. thanks.